src/request.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-console */
+const http = require('http');
+
+const BASE = 'http://localhost:5700';
```

Comment:
The BASE variable is hardcoded. It would be better to store this in a configuration file or environment variable for better flexibility.